admin_site/scripts/install.sh

```diff
@@ -26,23 +26,19 @@ then
 fi
 
 rm -rf $BIBOS_VIRTUAL_ENV
-virtualenv -p python3 $BIBOS_VIRTUAL_ENV
+python3 -m venv $BIBOS_VIRTUAL_ENV
 source $BIBOS_VIRTUAL_ENV/bin/activate
 
 DIR=$(dirname ${BASH_SOURCE[0]})
-PYTHON_PACKAGES=$(cat "$DIR/PYTHON_DEPENDENCIES")
-
-for  package in "${PYTHON_PACKAGES[@]}"
-do
-    pip install $package
-
-    RETVAL=$?
-    if [ $RETVAL -ne 0 ]; then
-        echo "" 1>&2
-        echo "ERROR: Unable to install Python package <$package>." 1>&2
-        echo -n "Please check your network connection. " 1>&2
-        echo "A remote server may be down - please retry later. " 1>&2
-        echo "" 1>&2
-        exit -1
-    fi
-done
+
+pip install -r ${DIR}/requirements.txt
+
+RETVAL=$?
+if [ $RETVAL -ne 0 ]; then
+    echo "" 1>&2
+    echo "ERROR: Unable to install Python package <$package>." 1>&2
```

Comment:
Without the loop construct, this will no longer work (since `$package` doesn't exist anywhere).

Reply:
Yeah. And in all cases, `pip` will have output something.